packages/xchain-cosmos/src/client.ts

```diff
@@ -119,7 +126,7 @@ class Client extends BaseXChainClient implements CosmosClient, XChainClient {
   }
 
   getSDKClient(): CosmosSDKClient {
-    return this.sdkClients.get(this.network) || TESTNET_SDK
+    return this.sdkClients.get(this.network) || TESTNET_SDK()
```

Comment:
Alternative value of `|| TESTNET_SDK()` is an artifact from a previous version. It can be removed it by changing `private sdkClients: Map<Network, CosmosSDKClient>` to `private sdkClients: Record<Network, CosmosSDKClient>`.

Reply:
Fixed in #599